src/pages/workspaces/workspace/analysis/modals/persistent-disk-controls.ts

```diff
@@ -77,33 +102,39 @@ export const PersistentDiskSection = ({ diskExists, computeConfig, updateCompute
       label({ style: computeStyles.label }, ['Persistent disk']),
       div({ style: { marginTop: '0.5rem' } }, [
         'Persistent disks store analysis data. ',
-        h(Link, { onClick: handleLearnMoreAboutPersistentDisk }, ['Learn more about persistent disks and where your disk is mounted.'])
+        h(Link, {
+          onClick: () => handleLearnMoreAboutPersistentDisk({ setViewMode })
+        }, ['Learn more about persistent disks and where your disk is mounted.'])
       ]),
       div({ style: { ...gridStyle, gridGap: '1rem', gridTemplateColumns: '15rem 5.5rem', marginTop: '0.75rem' } }, [
-        diskExists ?
-          h(TooltipTrigger, {
-            content: [
-              'You already have a persistent disk in this workspace. ',
-              'Disk type can only be configured at creation time. ',
-              'Please delete the existing disk before selecting a new type.'
-            ],
-            side: 'bottom'
-          }, [h(PersistentDiskType, { diskExists, computeConfig, updateComputeConfig })]) : h(PersistentDiskType, { diskExists, computeConfig, updateComputeConfig }),
+        // TODO: we inconsistently use GCP and Gcp, once cloudPlatform is typed, make stronger comparison here
+        ['GCP', 'Gcp'].includes(cloudPlatform) ? diskType({ persistentDiskExists, computeConfig, updateComputeConfig }) : false,
```

Comment:
Once you enforce the type on `cloudPlatform`, this TODO should trivially resolve